lang/sv.js

```diff
@@ -7,31 +7,33 @@ export default {
 	"components.calendar.selected": "Markerad.",
 	"components.calendar.show": "Visa {month}",
 	"components.dialog.close": "Stäng dialogrutan",
-	"components.form-element.defaultError": "{label} är ogiltigt.",
+	"components.form-element.defaultError": "{label} är ogiltig.",
 	"components.form-element.defaultFieldLabel": "Fält",
-	"components.form-element.input.email.typeMismatch": "",
-	"components.form-element.input.number.rangeOverflow": "",
-	"components.form-element.input.number.rangeUnderflow": "",
-	"components.form-element.input.url.typeMismatch": "",
+	"components.form-element.input.email.typeMismatch": "{label} is not a valid email",
+	"components.form-element.input.number.rangeOverflow": "Number must be lower than {max}",
+	"components.form-element.input.number.rangeUnderflow": "Number must be higher than {min}",
+	"components.form-element.input.url.typeMismatch": "{label} is not a valid URL",
 	"components.form-element.valueMissing": "{label} krävs.",
-	"components.form-error-summary.errorSummary": "",
+	"components.form-error-summary.errorSummary": "There {count, plural, one { was 1 error } other { were { count } errors }} found in the information you submitted",
 	"components.input-date-range.endDate": "Slutdatum",
-	"components.input-date-range.errorBadInput": "",
+	"components.input-date-range.errorBadInput": "{startLabel} must be before {endLabel}",
 	"components.input-date-range.startDate": "Startdatum",
 	"components.input-date-time-range.endDate": "Slutdatum",
 	"components.input-date-time-range.startDate": "Startdatum",
 	"components.input-date-time.date": "Datum",
 	"components.input-date-time.time": "Tid",
 	"components.input-date.clear": "Rensa",
-	"components.input-date.errorMaxDateOnly": "",
-	"components.input-date.errorMinDateOnly": "",
-	"components.input-date.errorOutsideRange": "Datum måste vara mellan {minDate} och {maxDate}",
+	"components.input-date.errorMaxDateOnly": "Datumet måste vara före {maxDate}",
+	"components.input-date.errorMinDateOnly": "Datumet måste vara efter {minDate}",
+	"components.input-date.errorOutsideRange": "Datumet ska vara mellan {minDate} och {maxDate}",
```

Comment:
went from "Dates must be between {minDate} and {maxDate}" to "The date should be between {minDate} and {maxDate}". Seems fine.